extensions/extensions-docker/aissemble-hive/aissemble-hive-service/pom.xml

```diff
@@ -15,4 +15,57 @@
 
     <packaging>orphedomos</packaging>
 
+    <properties>
+        <dockerbuild.jars.directory>target/dockerbuild/jars/</dockerbuild.jars.directory>
+        <version.hive.metastore>4.0.0</version.hive.metastore>
+        <version.hadoop>3.4.0</version.hadoop>
```

Comment:
I: Taking the opportunity to bump the Hadoop version for this service.  Out of sync with Spark (which uses 3.3.4) as that version is no longer available on https://dlcdn.apache.org/hadoop/common/.

Reply:
I would like to float the idea of using the `current` version rather than pegging a specific version, but I can see both pros and cons to this approach.  Namely, it becomes non-obvious what version we're actually on.